pom.xml

```diff
@@ -285,7 +285,7 @@
             </goals>
             <configuration>
               <resourceBundles>
-                <resourceBundle>org.eclipse.jetty.toolchain:jetty-artifact-remote-resources:1.1</resourceBundle>
+                <resourceBundle>org.eclipse.jetty.toolchain:jetty-artifact-remote-resources:1.2-SNAPSHOT</resourceBundle>
```

Comment:
This can be `1.2` final release (now available)

Reply:
@joakime thanks for the release!